### PR TITLE
Add documentation of header and footer for Select component

### LINKED ISSUE #2520: Add documentation of `header` and `footer` for `Select` component
### Subject

Select

### Description

#2519 

Add the functionality added by this PR to the document.

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by kaehehehe**
Closes #2520

## Description

Add header and footer prop descriptions to Select component documentation.

## Current behavior (updates)

https://github.com/user-attachments/assets/f7021e97-d416-4c62-a172-1009474ff1dd

### CODE REVIEW

File: docs/contents/components/forms/select/props.ja.mdx
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/props.ja.mdx:1]: This file is generated automatically and does not need to be changed.

File: docs/contents/components/forms/select/props.mdx
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/props.mdx:1]: This file is generated automatically and does not need to be changed.

File: docs/contents/components/forms/select/index.ja.mdx [hunk 1]
```diff
@@ -73,6 +73,32 @@ return <Select placeholder="キャラクターを選択" items={items} />
 </Select>
 ```
 
+### ヘッダーを追加する
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/index.ja.mdx:76]: Change "ヘッダーを追加する" to "ドロップダウンにヘッダーを追加する".

File: docs/contents/components/forms/select/index.mdx [hunk 2]
```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
 </Select>
 ```
 
+### Header
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/index.mdx:76]: Change "Header" to "Add header to a dropdown".

File: docs/contents/components/forms/select/index.mdx [hunk 3]
```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
...
+  placeholder="キャラクターを選択"
+  header={<header>Header added here</header>}
+>
+  <Option value="孫悟空">孫悟空</Option>
+  <Option value="ベジータ">ベジータ</Option>
+  <Option value="フリーザ">フリーザ</Option>
+</Select>
+```
+
+### Footer
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/index.mdx:89]: Change "Footer" to "Add footer to a dropdown".

File: docs/contents/components/forms/select/index.ja.mdx [hunk 4]
```diff
@@ -73,6 +73,32 @@ return <Select placeholder="キャラクターを選択" items={items} />
...
+  <Option value="フリーザ">フリーザ</Option>
+</Select>
+```
+
+### フッターを追加する
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  footer={<footer>ここにフッターが追加されます</footer>}
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/index.ja.mdx:94]: Change

```jsx
footer={<footer>ここにフッターが追加されます</footer>}
```

to

```
footer={<Center pb="2" px="3">ここにフッターが追加されます</Center>}
```

File: docs/contents/components/forms/select/index.mdx [hunk 5]
```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
...
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  footer={<footer>Footer added here</footer>}
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/index.mdx:94]: Change

```jsx
footer={<footer>Footer added here</footer>}
```

to

```jsx
footer={<Center pb="2" px="3">Footer added here</Center>}
```

File: docs/contents/components/forms/select/index.mdx [hunk 6]
```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
 </Select>
 ```
 
+### Header
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  header={<header>Header added here</header>}
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/select/index.mdx:81]: Change

```jsx
header={<header>Header added here</header>}
```

to

```jsx
header={<Center pt="2" px="3">Header added here</Center>}
```

File: docs/contents/components/forms/select/props.ja.mdx
Comment (kaehehehe) [reply docs/contents/components/forms/select/props.ja.mdx:1]: [257af9f](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/257af9f465a9a6e557c9fd687e256072af37c0aa)

File: docs/contents/components/forms/select/props.mdx
Comment (kaehehehe) [reply docs/contents/components/forms/select/props.mdx:1]: [257af9f](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/257af9f465a9a6e557c9fd687e256072af37c0aa)

File: docs/contents/components/forms/select/index.ja.mdx (hunk 1 above)
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.ja.mdx]: [d17fba8](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/d17fba86374ba91f3cd89c73516eea6942b57e3b)

File: docs/contents/components/forms/select/index.ja.mdx [hunk 7]
```diff
@@ -73,6 +73,32 @@ return <Select placeholder="キャラクターを選択" items={items} />
...
+  placeholder="キャラクターを選択"
+  header={<header>ここにヘッダーが追加されます</header>}
+>
+  <Option value="孫悟空">孫悟空</Option>
+  <Option value="ベジータ">ベジータ</Option>
+  <Option value="フリーザ">フリーザ</Option>
+</Select>
+```
+
+### フッターを追加する
```
[the remark this answers is not in the document]
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.ja.mdx]: [d17fba8](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/d17fba86374ba91f3cd89c73516eea6942b57e3b)

File: docs/contents/components/forms/select/index.ja.mdx [hunk 8]
```diff
@@ -73,6 +73,32 @@ return <Select placeholder="キャラクターを選択" items={items} />
 </Select>
 ```
 
+### ヘッダーを追加する
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  header={<header>ここにヘッダーが追加されます</header>}
```
[the remark this answers is not in the document]
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.ja.mdx]: [d17fba8](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/d17fba86374ba91f3cd89c73516eea6942b57e3b)

File: docs/contents/components/forms/select/index.ja.mdx (hunk 4 above)
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.ja.mdx]: [d17fba8](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/d17fba86374ba91f3cd89c73516eea6942b57e3b)

File: docs/contents/components/forms/select/index.mdx (hunk 6 above)
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.mdx]: [f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)

File: docs/contents/components/forms/select/index.mdx (hunk 5 above)
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.mdx]: [f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)

File: docs/contents/components/forms/select/index.mdx (hunk 3 above)
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.mdx]: [f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)

File: docs/contents/components/forms/select/index.mdx (hunk 2 above)
Comment (kaehehehe) [reply docs/contents/components/forms/select/index.mdx]: [f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)

**Comment by kaehehehe** [comment]
Thanks for the feedback! I've changed it! :) @hirotomoyamada

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**